### PR TITLE
[PhpUnitBridge] Add keyword `dev` to leverage composer hint

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

File: src/Symfony/Bridge/PhpUnit/composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "symfony/phpunit-bridge",
     "type": "symfony-bridge",
     "description": "Provides utilities for PHPUnit, especially user deprecation notices management",
-    "keywords": [],
+    "keywords": ['dev'],
```
Comment (norkunas) [inline src/Symfony/Bridge/PhpUnit/composer.json:5]: Are single quotes valid json?

Comment (OskarStark) [reply src/Symfony/Bridge/PhpUnit/composer.json:5]: No, but @fabpot fixed it while merging in e86d26abd07187d5ec5b2d780cfd1d872d57fe2a

Comment (fabpot) [reply src/Symfony/Bridge/PhpUnit/composer.json:5]: Not yet :)
I fixed it just after the merge when I realized that ;)

**Pull request merged**